src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
```

Comment:
It's unusual to define a default for a callback like `onPageChange`. If this prop is required for the component to function properly, it should not have a default that does nothing. This can mask bugs where the callback is not provided. Consider making the prop mandatory or handling the case where it's not provided more explicitly.